2.ensemble-z-analysis/scripts/z_sweep_jobs_submit.py

```diff
@@ -108,7 +102,7 @@
 walltime = config_df.loc['walltime']['assign']
 
 # Set default parameter combination
-conda = ['conda', 'activate', 'tybalt-gpu']
+conda = ['source', 'activate', 'tybalt-gpu', '&&']
```

Comment:
why the double ampersand?

Reply:
when running on pmacs I need to squash everything into one command - the ampersand allows me to activate the conda environment and then run the command. This is not necessary in the local setting